server/Idp/IDP.Domain/Models/Idp.cs

```diff
@@ -13,10 +13,6 @@ public class Idp
         [StringLength(200)]
         public string? Name { get; set; }
```

Comment:
The name is already there in the ApplicationUser table. Why did you add it again here?